app/actors/hyku_addons/actors/related_identifier_actor.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+module HykuAddons
+  module Actors
+    class RelatedIdentifierActor < Hyrax::Actors::AbstractActor
+      def create(env)
+        add_related_identifier_data(env) && next_actor.create(env)
+      end
+
+      def update(env)
+        add_related_identifier_data(env) && next_actor.update(env)
+      end
+
+      private
+
+        def add_related_identifier_data(env)
+          return unless env.curation_concern.class.to_s.include? "Redlands"
+          identifier_hash = JSON.parse(env.attributes[:related_identifier].first).first if env.attributes[:related_identifier]
+          env.attributes[:related_identifier] = [add_related_metadata(identifier_hash)]
```

Comment:
I like using `Array.wrap` when I try to ensure I convert something to an array because 1) Explicitly tells what it is doing 2) If what I am wrapping is already an array, it doesn't include it in yet another array:
```suggestion
          env.attributes[:related_identifier] = Array.wrap(add_related_metadata(identifier_hash))
```